spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewriteDataFilesProcedure.java

```diff
@@ -133,6 +133,43 @@ public void testRewriteDataFilesWithSortStrategy() {
     assertEquals("Data after compaction should not change", expectedRecords, actualRecords);
   }
 
+  @Test
+  public void testRewriteDataFilesWithZOrder() {
+    createTable();
+    // create 10 files under non-partitioned table
+    insertData(10);
+    List<Object[]> expectedRecords = currentData();
+
+    // set z_order = c1,c2
+    List<Object[]> output = sql(
+        "CALL %s.system.rewrite_data_files(table => '%s', " +
+        "strategy => 'zorder', sort_order => 'c1,c2')",
```

Comment:
@RussellSpitzer, what do you think about this work-around?

i would like to see something like `c3, zorder(c1, c2)` supported, but I don't know if that's going to be too complicated.

Reply:
I thought we were going to have the strategy just be sort. After that change, I think it's ok to special case zorder support at the moment and eventually support expressions like you are describing.

Reply:
Sorry, I am confused. 
Is the current implementation of `strategy => 'zorder', sort_order => 'c1,c2'` is ok or do we need to change this?
If we need to change this, can you mention an example of what the arguments look like?

Reply:
Strategy = sort, sort_order = zorder(c1, c2)